### PR TITLE
Feature update playground

### PR DESCRIPTION
**Pull request by MeepMr**
Updated the Demo-Website to the newest version
Integrated the Demo-Website into the production-version
Demo-Website and production-version are now only separated by two environment-variables

### CODE REVIEW

File: .github/workflows/release-demo-website.yml [hunk 1]
```diff
@@ -19,13 +19,33 @@ jobs:
...
-      # Runs a single command using the runners shell
-      - name: Run a one-line script
-        run: echo Hello, world!
-
-      # Runs a set of commands using the runners shell
-      - name: Run a multi-line script
-        run: |
-          echo Add other actions to build,
-          echo test, and deploy your project.
+      - name: Set up JDK 1.8
```
Comment (MeepMr) [inline .github/workflows/release-demo-website.yml:22]: Check if we still need the old configuration

File: frontend/src/app/demo/demo-home/demo-home.component.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    <button class="main-button button-bag-layout-inverted" mat-stroked-button [matMenuTriggerFor]="documentationMenu" i18n="@@component_demoHome_documentation">Dokumentation</button>
+
+    <span class="line-break hidden-s-up"></span>
+
+    <button class="main-button button-bag-layout-inverted" mat-stroked-button i18n="@@component_demoMainView_playground" (click)="openPlayground()">Playground</button>
+
+  </div>
+
+
+<!--  <div class="wow animate__fadeInDown" data-wow-duration="1s">-->
```
Comment (MeepMr) [inline frontend/src/app/demo/demo-home/demo-home.component.html:21]: Auskommentiertes HTML entfernen

File: frontend/src/environments/environment.test.ts [hunk 3]
```diff
@@ -2,5 +2,6 @@ import { Environment } from '../app/shared/model/environment/environment';
 
 export const environment: Environment = {
   production: true,
+  playground: true,
```
Comment (MeepMr) [inline frontend/src/environments/environment.test.ts:5]: Playground im Test nicht aktivieren

File: frontend/src/config.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+// By default, Klaro will load the config from  a global "klaroConfig" variable.
+// You can change this by specifying the "data-config" attribute on your
+// script take, e.g. like this:
+// <script src="klaro.js" data-config="myConfigVariableName" />
+// You can also disable auto-loading of the consent notice by adding
+// data-no-auto-load=true to the script tag.
+var beschreibungtext = 'Die BROCKHAUS AG verwendet Cookies, um Ihnen ein möglichst optimales und auf Ihre Bedürfnisse zugeschnittenes Webseiten-​Erlebnis zu bieten. Dazu zählen Cookies, die für den reibungslosen Betrieb der Seite und deren Sicherheit notwendig sind sowie solche, die zu anonymen Statistikzwecken genutzt werden. Sie können selbst entscheiden, welche der Kategorien Sie erlauben möchten.';
+if (window.innerWidth <= 625) {
+    beschreibungtext = '';// 'Die BROCKHAUS AG verwendet Cookies, um Ihnen ein möglichst optimales Webseiten-​Erlebnis zu bieten.';
```
Comment (INoTime) [inline frontend/src/config.js:9]: For what is the comment?

File: frontend/src/app/okr-units/okr-unit-card/okr-unit-card.component.html [hunk 5]
```diff
@@ -39,7 +39,9 @@
       <span i18n="@@component_unitCard_editName">Namen bearbeiten</span>
     </button>
 
-    <button [disabled]=!isCurrentUserAdmin mat-stroked-button (click)="$event.stopPropagation(); deleteCompany()"
+<!--      [disabled]=!isCurrentUserAdmin-->
```
Comment (MeepMr) [inline frontend/src/app/okr-units/okr-unit-card/okr-unit-card.component.html:42]: commented code

**Review by INoTime — APPROVED** [review]

**Review by MeepMr — COMMENTED** [review]
Delete frontend/src/app/core/auth/components/redirect/redirect.component.ts

**Review by McJanjo — APPROVED** [review]

**Review by McJanjo — APPROVED** [review]

**Review by cptmino — APPROVED** [review]

**Review by INoTime — APPROVED** [review]

**Pull request merged**